### PR TITLE
[RELEASE] 0.15.28

### PR DESCRIPTION
**Pull request by billdirks**
release prep for 0.15.28

### CODE REVIEW

**Comment by ghost** [comment]
<details>
<summary><b>👇 <a href="https://app.codesee.io/r/reviews?pr=6183&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations">Click on the image</a> for a new way to code review</b>
</summary>

  - Make big changes easier — review code in small groups of related files
  - Know where to start — see the whole change at a glance
  - Take a code tour — explore the change with an interactive tour
  - Make comments and review — all fully sync’ed with github

    [Try it now!](https://app.codesee.io/r/reviews?pr=6183&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations)
</details>

[Review these changes using an interactive CodeSee Map](https://app.codesee.io/r/reviews?pr=6183&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations)

#### Legend
CodeSee Map Legend

**Review by alexsherstinsky — APPROVED** [review]
LGTM -- thanks!

File: docs/changelog.md [hunk 1]
```diff
@@ -2,6 +2,29 @@
...
+* [BUGFIX] Ensure that spaces are recognized in named snippets (#6172)
+* [DOCS] Clarify wording for interactive mode in databricks (#6154)
+* [DOCS] fix source activate command (#6161) (thanks @JGrzywacz)
+* [DOCS] Update version in `runtime.txt` to fix breaking Netlify builds (#6181)
+* [DOCS] Clean up snippets and line number validation in docs (#6142)
+* [MAINTENANCE] Add Enums for renderer types (#6112)
+* [MAINTENANCE] Minor cleanup in preparation for Validator refactoring into separate concerns (#6155)
+* [MAINTENANCE] add the internal `GE_DATA_CONTEXT_ID` env var to the docker file (#6122)
+* [MAINTENANCE] Rollback setting GE_DATA_CONTEXT_ID in docker image. (#6163)
+* [MAINTENANCE] disable ge_cloud_mode when specified, detect misconfiguration (#6162) (thanks @rreinoldsc)
```
Comment (cdkini) [inline docs/changelog.md]: Please remove attribution to @rreinoldsc

File: docs_rtd/changelog.rst [hunk 2]
```diff
@@ -4,6 +4,30 @@
...
+* [BUGFIX] Ensure that spaces are recognized in named snippets (#6172)
+* [DOCS] Clarify wording for interactive mode in databricks (#6154)
+* [DOCS] fix source activate command (#6161) (thanks @JGrzywacz)
+* [DOCS] Update version in `runtime.txt` to fix breaking Netlify builds (#6181)
+* [DOCS] Clean up snippets and line number validation in docs (#6142)
+* [MAINTENANCE] Add Enums for renderer types (#6112)
+* [MAINTENANCE] Minor cleanup in preparation for Validator refactoring into separate concerns (#6155)
+* [MAINTENANCE] add the internal `GE_DATA_CONTEXT_ID` env var to the docker file (#6122)
+* [MAINTENANCE] Rollback setting GE_DATA_CONTEXT_ID in docker image. (#6163)
+* [MAINTENANCE] disable ge_cloud_mode when specified, detect misconfiguration (#6162) (thanks @rreinoldsc)
```
Comment (cdkini) [inline docs_rtd/changelog.rst]: Also here 🙇🏽

**Review by cdkini — CHANGES_REQUESTED** [review]

File: docs/changelog.md (hunk 1 above)
Comment (billdirks) [reply docs/changelog.md]: Done. Thanks for catching this!

File: docs_rtd/changelog.rst (hunk 2 above)
Comment (billdirks) [reply docs_rtd/changelog.rst]: Done.

**Review by cdkini — APPROVED** [review]

**Pull request merged**